src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -0,0 +1,157 @@
+
+That an interface has a legacy caller means that an instance can be called as a
+function. Currently, `HTMLEmbedElement` and `HTMLObjectElement` support this
+functionality. In Chrome 57 this ability was deprecated. Calling now throws an exception.
+
+This change brings Chrome in line with recent spec changes. The legacy behavior
+is not supported in Edge or Safari, and it is being
+[removed from Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=909656).
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/AiDZ7ru9mGg/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5715026367217664) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=663662)
+
+
+## Remove deprecated names for motion path properties
+
+Motion path CSS properties allow authors to animate any graphical object along an author-specified path. In climpliance with the spec, several properties were [implemented in Chrome 45](https://www.chromestatus.com/feature/6190642178818048). The names of these properties were changed in the spec in mid 2016. Chrome implemented the [new names in Chrome 55 and Chrome 56](https://www.chromestatus.com/feature/6390764217040896). Console deprecation warnings were also implemented. 
+
+In Chrome 58, the old versions are being removed. The affected properties and their new names are shown below.
+
+| Removed Property | Current Name |
+|------------------|--------------|
+| motion-path | offset-path |
+| motion-offset | offset-distance |
+| motion-rotation | offset-rotate |
+| motion | offset |
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/o1C5NzGf9Q0/discussion) 
+
+
+## Remove support for commonName matching in certificates
+
+[RFC 2818](https://tools.ietf.org/html/rfc2818) describes two methods to match a domain name against a certificate: using the available names within the `subjectAlternativeName` extension, or, in the absence of a SAN extension, falling back to the `commonName`. The fallback to the `commonName` was deprecated in RFC 2818 (published in 2000), but support remains in a number of TLS clients, often incorrectly.
+
+The use of the `subjectAlternativeName` fields leaves it unambiguous whether a certificate is expressing a binding to an IP address or a domain name, and is fully defined in terms of its interaction with Name Constraints. However, the `commonName` is ambiguous, and because of this, support for it has been a source of security bugs in Chrome, the libraries it uses, and within the TLS ecosystem at large.
+
+The compatibility risk is low. RFC 2818 has deprecated this for nearly two decades, and the baseline requirements (which all publicly trusted CAs must abide by) has required the presence of a `subjectAltName` since 2012. Mozilla Firefox already requires the `subjectAltName` for any newly issued publicly trusted certificates since [Firefox 48]( https://bugzilla.mozilla.org/show_bug.cgi?id=1245280 ).
```

Comment:
> `the baseline requirements`

Suggested Edit: `the [Baseline Requirements](https://cabforum.org/baseline-requirements-documents/)`